backend/src/main/java/gov/cdc/usds/simplereport/service/TestOrderService.java

```diff
@@ -345,15 +338,12 @@ public TestOrder addPatientToQueue(
     TestOrder newOrder = new TestOrder(patient, testFacility);
 
     AskOnEntrySurvey survey =
-        new AskOnEntrySurvey(
-            pregnancy,
-            symptoms,
-            noSymptoms,
-            symptomOnsetDate,
-            firstTest,
-            priorTestDate,
-            priorTestType,
-            priorTestResult);
+        AskOnEntrySurvey.builder()
+            .pregnancy(pregnancy)
+            .symptoms(symptoms)
+            .noSymptoms(noSymptoms)
+            .symptomOnsetDate(symptomOnsetDate)
+            .build();
```

Comment:
This is cool, I hadn't used the `@Builder` annotation before

Reply:
Yeah, my goal is eliminate the `new` keyword from the code, constructors are the worst since they are order dependent meaning it breaks if you alter the signature slightly and worse case it could lead to bad data. and you dont have to specify `null` fields with builders.